ckanext/harvest/logic/action/update.py

```diff
@@ -491,6 +496,88 @@ def harvest_jobs_run(context, data_dict):
 
     return []  # merely for backwards compatibility
 
+def send_error_mail(context, job_obj):
+
+    sql = 'select name from package where id = :source_id;'
+
+    model = context['model']
+
+    q = model.Session.execute(sql, {'source_id': job_obj.source.id})
+
+    for row in q:
+        harvest_name = str(row['name'])
+
+    ckan_site_url = config.get('ckan.site_url')
+    job_url = ckan_site_url + '/harvest/' + harvest_name + '/job/' + job_obj.id
+
+    msg = 'This is a failure-notification of the latest harvest job (' + job_url + ') set-up in ' + ckan_site_url + '.\n\n'
+
+    sql = '''select g.title as org, s.title as job_title from member m
+            join public.group g on m.group_id = g.id
+            join harvest_source s on s.id = m.table_id
+            where table_id = :source_id;'''
+
+    q = model.Session.execute(sql, {'source_id': job_obj.source.id})
+
+    for row in q:
+        msg += 'Organization: ' + str(row['org']) + '\n\n'
+        msg += 'Harvest Job Title: ' + str(row['job_title']) + '\n\n'
+
+    msg += 'Date of Harvest: ' + str(job_obj.created) + ' GMT\n\n'
+
+    out = {
+        'last_job': None,
+    }
+
+    out['last_job'] = harvest_job_dictize(job_obj, context)
+
+    msg += 'Records in Error: ' + str(out['last_job']['stats'].get('errored', 0)) + '\n'
+
+    obj_error = ''
+    job_error = ''
+
+    sql = '''select hoe.message as msg from harvest_object ho
+          inner join harvest_object_error hoe on hoe.harvest_object_id = ho.id
+          where ho.harvest_job_id = :job_id;'''
+
+    q = model.Session.execute(sql, {'job_id' : job_obj.id})
+    for row in q:
+        obj_error += row['msg'] + '\n'
+
+    sql = '''select message from harvest_gather_error where harvest_job_id = :job_id; '''
+    q = model.Session.execute(sql, {'job_id' : job_obj.id})
+    for row in q:
+        job_error += row['message'] + '\n'
+
+    if (obj_error != '' or job_error != ''):
+        msg += 'Error Summary\n\n'
+
+    if (obj_error != ''):
+        msg += 'Document Error\n' + obj_error + '\n\n'
+
+    if (job_error != ''):
+        msg += 'Job Errors\n' + job_error + '\n\n'
+
+    if obj_error or job_error:
+        msg += '\n--\nYou are receiving this email because you are currently set-up as Administrator for ' + config.get('ckan.site_title') + '. Please do not reply to this email as it was sent from a non-monitored address.'
+
+        # get recipients
+        email_recipients = config.get('email_to').split(',')
```

Comment:
`email_to` is a paster setting for getting the stacktraces. Let's have a specific setting for this called `ckan.harvest.status_mail.email_to` instead, and you can check if this is set on `harvest_jobs_run` rather than having a separate `ckan.harvest.status_mail.errored`